### ISSUE TITLE
PartNet models/performances with Pytorch

### ISSUE THREAD

**Issue by SilvioGiancola** [zeliu98/CloserLook3D#19]
Dear @zeliu98 , 
Congratulation for your ECCV work and thank you for the neat code. 
I was wondering why you are not sharing the PartNet models trained with Pytorch. Are you experimenting a gap in performances with respect to your Tensorflow implementation? Are you planning on releasing them soon?
Best,

**Comment by zeliu98** [comment]
Hi @SilvioGiancola 
Thanks for your interest in our work! 
Of course, we will sharing the PartNet models soon and we are still working on it, since the training has not finished yet. For temporary use, you can check this [ checkpoint](https://drive.google.com/file/d/1xKNsWN72nh3cJHsajQaBo-F3J34k1fQf/view?usp=sharing) for PosPool. It has been trained for 180 epochs and can achieve 49.3/52.4 mIoU for val/test data splits, which is close to the final scores reported.

**Comment by SilvioGiancola** [comment]
Great! Looking forward for the final models! 
I was trying to train you model myself (4x v100 32GB, batch size 16) but for some reason, I only reached a best testing performances of 52.009% mmpIoU (10x voting), after exactly 400 epochs. I set the default 600 epochs for training, so that was my best performing model. Also, after 507 epochs, the loss returned NaN and broke the whole training (returning ~5% mmpIoU after that).
Any tip to reproduce your results? How much variance did you experienced?

**Comment by zeliu98** [comment]
Hi @SilvioGiancola,
We have released the pytorch models for PartNet. As for the problem you mentioned, may be the learning rate is too large, since in our initial implementation, the learning rate is linearly increasing with the batch size.  We have fixed this in our latest code.
